### ISSUE TITLE
Substituting <script> tags in HTML with merged values

### ISSUE THREAD

**Issue by Adhara3** [samaxes/minify-maven-plugin#88]
I thing this plugin does a great job but this feature is, IMHO, missing and makes the merge feature less usable.
I saw that you created the bundle feature in 1.7.5 (great!) and that could be the starting point to substitute in the HTML files the old css/js script/link tags with the new ones.
In my scenario I have a single page app with only one HTML containing all the imports. I am using Angular JS which is best managed with several js files (50+).
It would be great to bundle all of them into a single file and change the script tag so that I only import that single file. That would improve performances a lot!

It is a nice to have, but nothing really critical since, while the HTTP request for 50+ files may be not ideal, for single page apps this is performed only once.

Thanks
Andrea

**Comment by sinoid** [comment]
+1 for this feature to be implemented as I'm facing exactly the same challenges Andrea mentioned, however in my scenario I don't have a single page app.

**Comment by stefan-niedermann** [comment]
+1 from me, too. i like the plugin but if i use it, my complete project depends on maven because i have to link a script file which does not exist until maven creates it.

**Comment by woodgoblin** [comment]
+1 also.
If it is possilbe, it will be great to do so.
I have a single-page app and all I need is to have over 50 scripts or only 2, depending on profile
